libs/containers/Sticker.lua

```diff
@@ -13,16 +13,12 @@ local Sticker, get = require('class')('Sticker', Snowflake)
 function Sticker:__init(data, parent)
 	Snowflake.__init(self, data, parent)
 	self.client._sticker_map[self._id] = parent
-	return self:_loadMore(data)
+	return nil
 end
 
 function Sticker:_load(data)
 	Snowflake._load(self, data)
-	return self:_loadMore(data)
-end
-
-function Sticker:_loadMore(data)
-	
+	return nil
```

Comment:
Returning `nil` from initializers is not necessary. It is used for "loading more" as a tail call optimization.